todo_app/helpers/trello_items.py

```diff
@@ -0,0 +1,75 @@
+import os
+import requests
+from .item_class_helper import Card_Item
+
+api_key = os.getenv('TRELLO_API_KEY')
+api_token = os.getenv('TRELLO_API_TOKEN')
+board_id = os.getenv('BOARD_ID')
+list_id_to_do = os.getenv('TO_DO_LIST_ID') 
+list_id_done = os.getenv('DONE_LIST_ID')
+
+def get_items():
```

Comment:
Similarly here, these environment variables might be better delayed until they're needed - the easiest approach is to just evaluate them later e.g.: in `get_items`:
```python
    try:
        response = requests.get(url = f"https://api.trello.com/1/boards/{os.getenv('BOARD_ID')}/cards?key={os.getenv('TRELLO_API_KEY')}&token={os.getenv('TRELLO_API_TOKEN')}")
```
On a real codebase we'd probably prefer a more classic OOP solution; wrap these functions in a class, and load the environment variables in a constructor, e.g. something like:
```python
class TrelloService():

    def __init__(self):
        self.api_key = os.getenv('TRELLO_API_KEY')
        ...

    def get_items(self):
        try:
            response = requests.get(url = f"https://api.trello.com/1/boards/{self.board_id}/cards?key={self.api_key}&token={self.api_token}")
        ...
```